fastlane/docs/README.md

```diff
@@ -38,6 +38,16 @@ before_all do |lane|
 end
 ```
 
+### `before_each` block
+
+This block will get executed *before* each lane switch. It supports the same actions as lanes.
```

Comment:
Actually let's move this to `Advanced.md`, since a majority of the fastlane users won't use this feature. It should be below the lane switching docs, since it's only useful when switching lanes, right?